pom.xml

```diff
@@ -193,35 +193,4 @@
     </pluginManagement>
   </build>
 
-  <profiles>
-    <profile>
-      <id>travis</id>
-      <build>
-        <plugins>
-          <plugin>
-            <groupId>org.eluder.coveralls</groupId>
-            <artifactId>coveralls-maven-plugin</artifactId>
-            <version>4.3.0</version>
-          </plugin>
-        </plugins>
-      </build>
-    </profile>
-    <profile>
-      <id>doclint-java8-disable</id>
```

Comment:
are we not using this?

Reply:
We are, it's already in the parent so no need to have it here again.

Reply:
ok, I see